frontend/public/components/marketplace/kubernetes-marketplace-subscribe.jsx

```diff
@@ -0,0 +1,169 @@
+            onChange={onChange}
+          />
+        </Firehose>
+        :<Dropdown
+          title={title}
+          id={id}
+          items={items}
+          dropDownClassName="dropdown--full-width"
+          onChange={onChange}
+        />}
+      </div>
+    );
+  }
+}
+class AdminSubscribe extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      form: {
+        dropdowns: {
+          namespace: null,
+          channel: null,
+          strategy: null
+        }
+      }
+    };
+  }
+
+  onChange(selected, id) {
+    this.setState((prevState) => {
+      const form = prevState.form;
+      form.dropdowns[id] = selected;
+      return {
+        form: form
+      };
+    });
+  }
+
+  render () {
+    const { item, close, subscribe } = this.props;
+    if (item === null) {
+      return null;
+    }
+    const { name, imgUrl, provider, description } = item;
+
+    const form = [
+      {
+        title: 'Target Namespace',
+        id: 'namespace',
+        help: 'Select a namespace to subscribe this operator to.',
+      },
+      {
+        title: 'Update Channel',
+        id: 'channel',
+        items: {
+          stable: 'Stable',
+          latest: 'Latest'
+        }
+      },
+      {
+        title: 'Update Strategy',
+        id: 'strategy',
+        items: {
+          automatic: 'Automatic',
+          manual: 'Manual'
+        }
+      }
+    ];
+    return (
+      <div className="co-marketplace-subscribe">
+        <div className="co-marketplace-subscribe--header">
+          <h1>Subscribe Service</h1>
+          <p className="co-help-text">Select the namespace where you want to make the application available and select channel and update strategy.</p>
+        </div>
+        <div className="co-marketplace-subscribe--form">
+          <div className="co-marketplace-subscribe--dropdowns">
+            { form &&
+              form.map((dropdown, index) => (
+                <DropdownElement
+                  key={`subscribe-dropdown-${dropdown.id}-${index}`}
+                  id={dropdown.id}
+                  title={dropdown.title}
+                  items={dropdown.items}
+                  help={dropdown.help}
+                  onChange={(selected) => this.onChange(selected, dropdown.id)}
+                />
+              ))
+            }
+          </div>
+          <div className="co-marketplace-subscribe--buttons">
+            <Button bsStyle="primary" className="btn-cancel" onClick={() => subscribe(this.state.form.dropdowns.namespace)}>
+              Subscribe
+            </Button>
+            <Button bsStyle="default" onClick={close}>
+              Cancel
+            </Button>
+          </div>
+        </div>
+        <div className="co-marketplace-subscribe--description">
+          <CatalogItemHeader
+            className="long-description-test"
+            iconImg={imgUrl}
+            title={name}
+            vendor={<span> {provider}</span>}
```

Comment:
This should be consistent with the details header:

```suggestion
            vendor={`${version} provided by ${provider}`}
```